### PR TITLE
multiple entry points first reorganisation 

### PR DESCRIPTION
**Pull request by wsc1**
This won't compile, but the layout
is much more palatable and inviting.

Looking for feedback about the layout of {README,Contributing,Port}.md, (Port.md is less complete
than the others)

### CODE REVIEW

**Comment by wsc1** [comment]
perhaps better to link to the fork at this point in the design discussion.

File: Port.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Ports
+
+## Status
+The status of ports is kept in the main module [README](README.md).
+
+## Overview
+ZikiChombo sio code has a porting process that is unconventional due to the
```
Comment (mewmew) [inline Port.md:7]:
```diff
-unconventional due to the nature sound
+unconventional due to the nature of sound
```

File: Port.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+nature sound support on host systems.  Usually, to port a system, on takes some
+desired cross-host functionality, such as a filesystem or user interface, and
+implements it the OS or OS/hardware combination of interest and that's that 
+(to make a long story short).
+
+For sound, the problem is that hosts usually have multiple different 
+software components as entry points to playing and capturing sound,
```
Comment (mewmew) [inline Port.md:14]:
```diff
-entry points to playing and capturing
+entry points for playing and capturing
```

File: Port.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+and moreover these entry points have some mutually exclusive 
+properties, such as lower bounds on latency, availability of 
+duplex synchronised by hardware audio clock, ability to be shared
+accross many users or applications, etc.
+
+Although the main cpu hardware architectures can certainly be a concern,
```
Comment (mewmew) [inline Port.md:20]:
```diff
-cpu
+CPU
```

**Review by mewmew — COMMENTED** [review]
Looks like a good start! We'll figure out the specifics by experimenting with the API design, to see if it works seamlessly across platforms.

File: Port.md (hunk 3 above)
Comment (wsc1) [reply Port.md:20]: thanks. 

First I need to make it at least compile and pass some tests.

Maybe by the end of the weekend, maybe during next week.

**Pull request closed without merging**